apps/meteor/ee/client/voip/modal/DialPad/PadButton.tsx

```diff
@@ -1,31 +1,42 @@
 import { Box, Button } from '@rocket.chat/fuselage';
 import React, { ReactElement } from 'react';
 
-const letters = ['+', '', 'ABC', 'DEF', 'GHI', 'JKL', 'MNO', 'PQRS', 'TUV', 'WXYZ'];
+import type { PadDigit } from './Pad';
+import { useLongPress } from './hooks/useLongPress';
 
 const PadButton = ({
 	children,
 	onClickPadButton,
+	onLongPressPadButton,
 }: {
-	children: string | number;
-	onClickPadButton: (digit: string | number) => void;
-}): ReactElement => (
-	<Button
-		m='8px'
-		pb='8px'
-		minWidth='28%'
-		display='flex'
-		flexGrow={1}
-		flexShrink={0}
-		flexDirection='column'
-		alignItems='center'
-		onClick={(): void => onClickPadButton(children)}
-	>
-		<Box fontSize='h2'>{children}</Box>
-		<Box fontSize='c1' color='info'>
-			{typeof children === 'number' && letters[children]}
-		</Box>
-	</Button>
-);
+	children: PadDigit;
+	onClickPadButton: (digit: PadDigit) => void;
+	onLongPressPadButton: (digit: PadDigit) => void;
+}): ReactElement => {
+	const { onClick, onMouseDown, onMouseUp, onTouchStart, onTouchEnd } = useLongPress(() => onLongPressPadButton(children), {
+		onClick: () => onClickPadButton(children),
+	});
```

Comment:
- First I believe we can improve readability destructuring the children array, like so `const [digit, longDigit] = children`. 
- Second I think you should return only the appropriate digit, instead of the tuple. Ex.: `onClickPadButton(digit)` and `onLongPressPadButton(longDigit)`. This way we don't need to share this implementation detail to other parts of the code.

Reply:
ok!